.travis.yml

```diff
@@ -1,10 +1,24 @@
 language: ruby
 rvm:
-  - 2.2
-  - 2.1
-  - 2.0
-  - 1.9
-  - jruby-19mode
+  - "2.3.1"
+  - "2.2.2"
+  - "2.1"
+  - "2.0"
+  - "1.9"
+  - "jruby-19mode"
```

Comment:
some of these older rubies could also probably be dropped as they're unsupported.